src/Files.App/UserControls/Widgets/RecentFilesWidget.xaml.cs

```diff
@@ -30,5 +32,37 @@ private void RecentFilesListView_RightTapped(object sender, RightTappedRoutedEve
 		{
 			ViewModel.BuildItemContextMenu(e.OriginalSource, e);
 		}
+
+		private async void RecentFilesListView_DragItemsStarting(object sender, DragItemsStartingEventArgs e)
+		{
+			var items = e.Items.OfType<RecentItem>().ToList();
+			if (items.Count > 0)
```

Comment:
The recent files widget doesn't support multiselect, so the only possibility is for a single item to be dragged.